rskj-core/src/main/java/co/rsk/core/bc/BlockResult.java

```diff
@@ -85,7 +85,7 @@ public BigInteger getPaidFees() {
         //TODO Fix Trie hash for receipts - doesnt match cpp
         Trie receiptsTrie = new TrieImpl();
 
-        if (receipts.isEmpty()) {
```

Comment:
This change was made as part of this same PR: https://github.com/rsksmart/rskj/pull/304/commits/19002c019392387e53dad0b31262ef99d6e4759c

Did you undo it for any reason or was it just a merge conflict?